jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/ContentProducer.java

```diff
@@ -59,17 +59,17 @@
     void checkMinDataRate();
 
     /**
-     * Get the byte count produced by the underlying {@link org.eclipse.jetty.server.Content.Reader}.
+     * Get the byte count produced by the underlying {@link Content.Source}.
      *
      * This call is always non-blocking.
      * Doesn't change state.
-     * @return the byte count produced by the underlying {@link org.eclipse.jetty.server.Content.Reader}.
+     * @return the byte count produced by the underlying {@link Content.Source}.
      */
     long getRawContentArrived();
```

Comment:
Now that this class isn't dealing with `Content`s but `Chunk`s this method should be renamed.  Maybe to `getRawBytesArrived()`?